exercise/templatetags/exercise.py

```diff
@@ -118,6 +134,45 @@ def submission_status(status):
     return Submission.STATUS[status]
 
 
+"""
+    Used for adding relevant classes to the
+    points_badge to enable textual status.
+"""
+@register.simple_tag(takes_context=True)
+def get_points_class(context):
+    html_class = set()
+    if context['difficulty'] == "-" and context['max'] <= 2:
+        html_class.add("aplus-points")
+        class_name = ""
+
+        if ((context['max'] == 2 and context['points'] == 2)
+        or context['max'] == 1 and context['points']== 1):
+            class_name = "accepted"
+        elif (context['max'] == 2 and context['points'] == 1):
+            class_name = "ok"
+        elif context['points'] == 0 and context['submitted']:
+            class_name = "rejected"
+
+        html_class.add(class_name)
+        context['class'] = class_name
+    return " ".join(html_class)
+
+
+#Adds the content of the points_badge.
+@register.simple_tag(takes_context=True)
+def get_points_text(context):
+    if context['difficulty'] == "-" and context['max'] <= 2:
+        class_name = context['class']
+        texts = {
+          "ok": _("OK"),
+          "accepted": _("Accepted"),
+          "rejected": _("Rejected"),
+          "": _("Missing"),
+        }
+        return texts[class_name]
```

Comment:
`texts.get(class_name, _("Unknown"))` is a bit more robust method. or `texts.get(class_name) or _("Unknown")` to be bit more optimal when running.

Reply:
Implemented the first option.